ocw/data_source/local.py

```diff
@@ -268,3 +268,38 @@ def load_file(file_path,
 
     return Dataset(lats, lons, times, values, variable=variable_name,
                    units=variable_unit, name=name, origin=origin)
+
+def load_multiple_files(data_info):
```

Comment:
I'm not the biggest fan of passing info this way. The config related stuff should be handled in the proper package and necessary parameters should be passed into this function independent of the nested config format. Otherwise this becomes dependent upon the format of a config file that is subject to change and it's not terribly helpful to call when not using output from the config file.

In other words
* This requires output from a parsed config file to function, and that's confusing.
* The format of the config file isn't specified here, so the user has no idea how to actually use this function unless they read the config related stuff. If they have to use the config stuff then this functionality should be in the config related code it would seem.
* If/when the format of the config file changes this is subject to break, which is just poorly encapsulating functionality it seems.

I think this would be much cleaner if this function had a defined interface that was independent of the config file instead of taking nested config output.